codegenerator/cli/templates/static/codegen/src/ContractInterfaceManager.res

```diff
@@ -20,23 +20,12 @@ let make = (
   let contractNameInterfaceMapping = Js.Dict.empty()
 
   contracts->Belt.Array.forEach(contract => {
-    let {name, abi} = contract
-    let interface = Ethers.Interface.make(~abi)
```

Comment:
With the change, we don't need to create the `Ethers.Interface` anymore. It was done at least on every getLogs request. Probably not a bottleneck, but definitely good to avoid it.